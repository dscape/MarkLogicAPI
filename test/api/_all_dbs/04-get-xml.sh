#!/bin/sh
curl -sH "Accept: application/xml" localhost:6984/_all_dbs \
  | grep '>Documents</database>' | wc
