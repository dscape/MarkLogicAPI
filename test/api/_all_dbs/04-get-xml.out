<databases>
  <database id="12803784884617966720">Security</database>
  <database id="4754745607263771025">Schemas</database>
  <database id="11566888833277679620">Documents</database>
  <database id="15153962187444946487">App-Services</database>
  <database id="11552807500255451800">Last-Login</database>
  <database id="16102477973380615723">Triggers</database>
  <database id="11537455809792408525">Modules</database>
  <database id="13510060154442378446">Fab</database>
</databases>
 